.github/workflows/process_release.yml

```diff
@@ -218,6 +218,33 @@ jobs:
         asset_path: ./assets/antrea-multicluster-member.yml
         asset_name: antrea-multicluster-member.yml
         asset_content_type: application/octet-stream
+    - name: Upload antrea-agent-linux-x86_64
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ github.event.release.upload_url }}
+        asset_path: ./assets/antrea-agent.exe
```

Comment:
is the path correct? The job name says it's for linux.

Reply:
thanks for catching this. This is the old binary name (before this PR, we only generated one binary, for windows). I amended my commit.